pkg/access_log_server/access_log_server.go

```diff
@@ -0,0 +1,127 @@
+package access_log_server
```

Comment:
I'm not sure about the package and module name.
Its functionality is strictly bound to Envoy proxy, so maybe it should be called that way?

And if not, and we are trying to keep it generic, I'd drop the `access log` because it can be any protobuff or REST message. Something like `HttpIngester` (just a quick brainstorming)

Reply:
I would go with `envoy_access_log_server` then.

Reply:
Ok, my only concern is if the word `server` isn't somehow misleading since it's just another module with HTTP interface.
`listener` or `receiver` comes to my mind but maybe the `server` is ok... WDYT?

Reply:
To me, `server` makes sense. Another possibility would be [envoy_access_log_service](https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/accesslog/v2/als.proto).

But tbh, I don't have particulary strong opinion about this one.

Reply:
`envoy_access_log_service` it is, as agreed within offline discussion.